options.go

```diff
@@ -177,6 +183,25 @@ func IdentityResolverFunc(resolver IdentityResolver) Option {
 	}
 }
 
+// Installs RingChecksumStatPeriod, which determines the period between
```

Comment:
Also, that would fix this minor golint note:

```
$ golint options.go 
options.go:186:1: comment on exported function RingChecksumStatPeriod should be of the form "RingChecksumStatPeriod ..."
```

Which is a result of this convention:

_"The first sentence should be a one-sentence summary that **starts with the name being declared**."_
https://golang.org/doc/effective_go.html#commentary

Yay, code formatting conventions! :dancers: